backEnd/api/routes/Employer.Login.router.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const utils = require('../utils/authUtils');
+const Employer = require('../models/Employer/Employer.model');
+const { RouterFactory } = require('express-router-factory');
+
+const router = express.Router();
+
+const RF = new RouterFactory(router, Employer);
+
+RF.POST('/', "auth" ,loginEmployer);
+
+
+/**
+ * ROUTER HANDLERS: handle endpoints
+ */
+
+
+function loginEmployer(req, res, next) {
+  const { companyEmail, password } = req.body;
+  //console.log("email", companyEmail, "password", password)
```

Comment:
This doesn't need to be here any more. We should keep console logs used for development out of the master branch